.gitignore

```diff
@@ -1 +1,3 @@
 /node_modules
+.idea/
```

Comment:
this should go in `~/.gitignore`, not the project specific `.gitignore`.

you might need to set this in your `~/.gitconfig` file:

```ini
[core]
	excludesfile = ~/.gitignore
```